docs/UserGuide.md

```diff
@@ -61,139 +77,130 @@ Adds a new class to the list of classes.
 
 * CLASS_NAME must be unique (cannot have two classes with the same name)
 
-**Example:**
-add /c cs2103t
+**Examples:**
+* `add /c cs2103t`
 
 **Successful Output:**
-“CLASS_NAME” has been added
+* “CLASS_NAME” has been added
```

Comment:
Perhaps you can remove the quotation marks for CLASS_NAME, I feel that a user would assume the class name to be CLASS_NAME based on the Command Format and so it would be fine to be CLASS_NAME has been added as the successful output (similar to line 165) 😶 

I think standardising the English throughout the document can be tiring so I tried looking through for similar mistakes and found them at line 87, 100, 108, 111, 127, 130. No guarantee that I did not miss out any but I hope this helps! 😄